core/controllers/story_viewer.py

```diff
@@ -57,11 +57,7 @@ def get(self, story_id):
 
         ordered_node_dicts = [
             node.to_dict() for node in story.story_contents.get_ordered_nodes()
-            # TODO(aks681): Once the story publication is done, add a check so
```

Comment:
This functionality is present now and stories with unpublished explorations cannot be published.